src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy

```diff
@@ -21,11 +24,63 @@ import spock.lang.Narrative
 import java.util.concurrent.TimeUnit
 
 @Narrative("Verify scenarios around replugging ISLs between different switches/ports.")
+@Tags([TOPOLOGY_DEPENDENT])
 class IslReplugSpec extends HealthCheckSpecification {
 
     @Value('${opentsdb.metric.prefix}')
     String metricPrefix
 
+    def "Roundtrip ISL status changes to MOVED when replugging it into another switch"() {
+        given: "A connected a-switch link, roundtrip-enabled"
+        def isl = topology.islsForActiveSwitches.find {
+            it.aswitch?.inPort && it.aswitch?.outPort &&
+                    it.srcSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD) &&
+                    it.dstSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)
+        }
+        assumeTrue("Wasn't able to find enough of required a-switch links with round-trip", isl.asBoolean())
+
+        and: "A non-connected a-switch link"
+        def notConnectedIsl = topology.notConnectedIsls.find {
+            it.srcSwitch != isl.srcSwitch &&
+                    it.srcSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)
```

Comment:
Why do you need `NOVIFLOW_COPY_FIELD`? Do you need support RTL for the non-connected link?
If so, let's add this info into the step description

Reply:
changed to `and: "A non-connected a-switch link with round-trip support"`